bin/git-ignore-io

```diff
@@ -0,0 +1,124 @@
+#/bin/sh
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n"`
```

Comment:
#Today I Learn#  Maybe it is another difference between GNU coreutils and the BSD one.